lasagne/layers/pool.py

```diff
@@ -755,3 +756,161 @@ def get_output_shape_for(self, input_shape):
 
     def get_output_for(self, input, **kwargs):
         return self.pool_function(input.flatten(3), axis=2)
+
+
+def pooling_fixed_output_size(inputs, output_size, pooling_op=T.max):
```

Comment:
I agree that the name is not the best. It is in fact one possible implementation of pooling with fixed output sizes. 

What do you mean by moving this into a separate function?